src/block/button/style.scss

```diff
@@ -1,110 +1,12 @@
 @import "common";
 
+// We have a block group, and that has the bottom margin.
+.stk-block-button {
+	--stk-block-margin-bottom: 0;
+}
+
 .stk-block.stk-block-button {
 	width: auto;
-}
-.stk-block-button {
 	margin: 0;
 	text-align: center;
```

Comment:
I checked this, if I remove them they don't seem to do anything. Do we still need these styles?